build/deploy/mkimage.sh

```diff
@@ -13,19 +13,16 @@ set -ex
 cd -P "$(dirname $0)"
 DIR=$(pwd -P)
 
-rm -rf resources ui cockroach
-mkdir -p build
+rm -rf build && mkdir -p build
 docker run -v "$(pwd)/build":/build "cockroachdb/cockroach-dev" shell "cd /cockroach && \
   rm -rf /build/*
   make clean build testbuild >/dev/null 2>&1 && \
   find . -name '*.test' -type f -printf "\"/build/%h \"" | xargs mkdir -p && \
   find . -name '*.test' -type f -exec mv {} "/build/{}" \; && \
-  cp -r resources /build/resources && \
-  cp -r ui /build/ui && \
+  cp -r {ui,resources} /build/ && \
```

Comment:
Never really took a look at this before and I'm mildly confused by what it is doing. You've mounted `${pwd}/build` from the host into `/build` in the container and are then copying the `ui` and `resources` directories. Why? I'm going to poke around so I might end up answering the question myself.